src/flavors/__tests__/fixtures.ts

```diff
@@ -51,6 +52,16 @@ export function createFixtures(
     class Example {
         /* Create one property of each type that is required. */
 
+        @IsArray({
```

Comment:
It's important to have examples using primitive and object types because these are handled differently.